boards/adafruit-clue/include/board.h

```diff
@@ -107,7 +107,8 @@ extern "C" {
 #define ILI9341_PARAM_NUM_LINES (240U)                          /**< Number of screen lines */
 #define ILI9341_PARAM_RGB       (1)                             /**< RGB configuration */
 #define ILI9341_PARAM_INVERTED  (1)                             /**< Inversion configuration */
-#define ILI9341_PARAM_ROTATION  (LCD_ROTATION_HORZ_FLIP)        /**< Rotation mode */
+#define ILI9341_PARAM_ROTATION  (LCD_ROTATION_VERT)             /**< Rotation mode */
+#define ILI9341_PARAM_OFFSET_X  (80)                            /**< Vertical rotation requires a 80 pixel offset */
```

Comment:
This seems a bit weird, are you able to correctly draw in the whole screen? the 80 offset makes me think it has something to do with the display being `240X320` and with the flip you want to treat it as `320x240`, but its strange that this is not handled by the driver.

If you are able to correctly draw to the whole display I'm ok as it is, although it still seems a bit weird to me.

Reply:
On the adafruit clue, the display is 240x240 and yes it draws correctly with the 80 pixels offset. I also tested this on the pinetime (also 240x240) and to rotate by 180°, I also had to add an offset (on Y this time).